.github/actions/setup-testground/action.yml

```diff
@@ -0,0 +1,35 @@
+name: start testground
+description: setup a local testground instance
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout testground
+      uses: actions/checkout@v2
+      with:
+        path: testground
+        repository: testground/testground
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: "1.16.x"
+
+    - name: Install testground
+      run: make install
+      working-directory: testground
+      shell: bash
+
+    - name: Run the daemon or configure the client
+      shell: bash
+      run: |
+        if [[ ! -z "${TESTGROUND_ENDPOINT}" ]]; then
+          mkdir -p ~/testground/;
+          cat <<EOF >> ~/testground/.env.toml
+
+          [client]
+          endpoint = "${TESTGROUND_ENDPOINT}"
+        EOF
+        else
+          testground daemon > testground.out 2> testground.err &
+        fi;
```

Comment:
Would it make sense to wait for the daemon to be ready here as well?

https://github.com/testground/sdk-rust/blob/f19ef9621e319b06f8def74a6f28b715feca7010/.github/workflows/ci.yml#L107-L115

Reply:
That make sense, thanks for the catch, fixed!